knowledge-base/chart-palette-mvvm.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Different color for each bar
+description: Apply different bar chart colors using MVVM.
+type: how-to
+page_title: Different Chart Palette for Bar Series using MVVM pattern.
+slug: net-maui-chart-palette-mvvm
+position: 
+tags: maui, chart, barseries
+ticketid: 1567000
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product Version</td>
+			<td>2.0.0</td>
+		</tr>
+		<tr>
+			<td>Product</td>
+			<td>Chart for .NET MAUI</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+Currently the only way to add a Chart Palette is by adding a Palette entry in XAML.
+To be able to set a different color to the Bar Series Chart by using a MVVM manner, a custom chart Palette has to be created in the ViewModel
```

Comment:
Currently the only way to add a Chart Palette is by adding a Palette entry in XAML. This article will show you how-to set a different color to the Chart Bar Series using MVVM manner. 
In general you have to create a custom chart palette in the ViewModel